plugin/slime.vim

```diff
@@ -44,26 +44,34 @@ endfunction
 " Tmux
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
+function! s:TmuxSocket()
+  " The socket path is the first value in the comma-separated list of $TMUX.
+  return split($TMUX, ',')[0]
```

Comment:
The destination terminal needs to run TMUX. The "source" terminal, where VIM runs, doesn't. In practice, I've used vim-slime in both configurations.

However the configuration can be obtained, I don't think it's a good idea to assume the (TMUX) variable exists.